packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -192,27 +404,46 @@ class CupertinoNavigationBar extends StatelessWidget implements ObstructingPrefe
 /// close button in case of a fullscreen dialog) to pop the current route if none
 /// is provided and [automaticallyImplyLeading] is true (true by default).
 ///
+/// The [largeTitle] widget will automatically be a title text from the current
+/// route if none is provided and [automaticallyImplyTitle] is true (true by
+/// default).
+///
+/// When [transitionBetweenRoutes] is true, this navigation bar will transition
+/// on top of the routes instead of inside it if the route being transitioned
+/// to also has a [CupertinoNavigationBar] or a [CupertinoSliverNavigationBar]
+/// with [transitionBetweenRoutes] set to true. If [transitionBetweenRoutes] is
+/// true, none of the [Widget] parameters can contain a key in its subtree since
```

Comment:
"none of" is plural, "its" is singular, so `its subtree->`their subtrees` or some such.

Reply:
presumably this should say GlobalKey, not just key

Reply:
that is, "a key" -> "any [GlobalKey]s"